paper-api/src/main/java/org/bukkit/structure/Structure.java

```diff
@@ -155,6 +155,27 @@ public interface Structure extends PersistentDataHolder {
     @ApiStatus.Experimental
     void place(@NotNull RegionAccessor regionAccessor, @NotNull BlockVector location, boolean includeEntities, @NotNull StructureRotation structureRotation, @NotNull Mirror mirror, int palette, float integrity, @NotNull Random random, @NotNull Collection<BlockTransformer> blockTransformers, @NotNull Collection<EntityTransformer> entityTransformers);
 
+    /**
+     * Place a structure in the world.
+     *
+     * @param location The location to place the structure at.
+     * @param placementOptions Options used to determine different settings applied to the structure.
+     * @param blockTransformers A collection of {@link BlockTransformer}s to apply to the structure.
+     * @param entityTransformers A collection of {@link EntityTransformer}s to apply to the structure.
+     */
+    @ApiStatus.Experimental
+    void place(@NotNull Location location, @NotNull PlacementOptions placementOptions, @NotNull Collection<BlockTransformer> blockTransformers, @NotNull Collection<EntityTransformer> entityTransformers);
```

Comment:
What is the point of this method, I am a bit confused here.
If the placement options already track the transformers for blocks and entities, this method seems problematic.